### DISCUSSION TITLE
Getting replies of a particular tweet

### DISCUSSION THREAD

**Discussion by AnkS4** [tweepy/tweepy#1890] [Questions] [answered]
To get the replies I am searching for mentions of the OP & checking it's in_reply_to_status_id, that way I am getting around 10 replies after setting 100 tweet limit.

Is there more efficient way to get the replies of a particular post?

```
replies = tweepy.Cursor(
    api.search_tweets,
    q = 'to:'+user.screen_name,
    since_id = tweet_id,
).items(100)

reply_list = []
for reply in replies:
    if reply.in_reply_to_status_id == tweet_id:
        reply_list.append(reply)

print(reply_list)
```

**Comment by Harmon758** [comment, marked as answer]
With Twitter API v2, you can search by [conversation ID](https://developer.twitter.com/en/docs/twitter-api/conversation-id).
